### PR TITLE
AAP-29296-A updated system requirements

### PR DESCRIPTION
**Pull request by emurtoug**
AAP-29296-A: child of [AAP-29296](https://issues.redhat.com/browse/AAP-29296)

Updated system requirements section of the Planning guide. See [[WIP] v2.5 Red Hat Ansible Automation Platform planning guide](https://docs.google.com/document/d/1tCXiQxr0WspctajD7Hxwji9zsaOaPMl-WhRsB1sF-qs/edit).

Files added: 
- ref-OCP-system-requirements.adoc
- ref-RPM-system-requirements.adoc
- ref-containerized-system-requirements.adoc

Files modified:
- assemblies/platform/assembly-system-requirements.adoc

### CODE REVIEW

File: downstream/modules/platform/ref-OCP-system-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+
+// [id="ref-OCP-system-requirements_{context}"]
+
+= System requirements for installing {PlatformName} on {OCPShort}
+
```
Comment (dcdacosta) [inline downstream/modules/platform/ref-OCP-system-requirements.adoc:6]: I wonder if you shouldn't make the titles here align with the titles of the different install guides? Suggest:
System requirements for installing on {OCPShort}

File: downstream/modules/platform/ref-containerized-system-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+
+// [id="ref-containerized-system-requirements_{context}"]
+
+= System requirements for containerized installation of {PlatformNameShort}
+
```
Comment (dcdacosta) [inline downstream/modules/platform/ref-containerized-system-requirements.adoc:6]: Suggest: System requirements for containerized installation

**Review by dcdacosta — COMMENTED** [review]
A few minor comments, otherwise, LGTM!

File: downstream/modules/platform/ref-containerized-system-requirements.adoc (hunk 2 above)
Comment (emurtoug) [reply downstream/modules/platform/ref-containerized-system-requirements.adoc:6]: Change made, thank you!

File: downstream/modules/platform/ref-OCP-system-requirements.adoc (hunk 1 above)
Comment (emurtoug) [reply downstream/modules/platform/ref-OCP-system-requirements.adoc:6]: Change made, thank you!

**Comment by michellemacrh** [comment]
LGTM!

**Pull request merged**